public/app/features/dashboard-scene/solo/SoloPanelPage.tsx

```diff
@@ -36,6 +36,16 @@ export function SoloPanelPage({ queryParams }: Props) {
     return <EntityNotFound entity="Panel" />;
   }
 
+  if (loadError) {
+    return (
+      <Box justifyContent={'center'} alignItems={'center'} display={'flex'} height={'100%'}>
+        <Alert severity="error" title="Failed to load dashboard">
```

Comment:
```suggestion
        <Alert severity="error" title={t(Failed to load dashboard)}>
```

We could add translation here :)